main/src/http/http_service.rs

```diff
@@ -1257,6 +1358,112 @@ fn construct_write_tsdb_points_json_request(req: &Bytes) -> Result<Vec<Line>, Ht
     Ok(tsdb_datapoints)
 }
 
+fn _try_parse_req_to_log_lines(
+    req: &Bytes,
+    table: String,
+    time_alias: Option<String>,
+    msg_alias: Option<String>,
+) -> Result<Vec<Line>, HttpError> {
+    let lines = simdutf8::basic::from_utf8(req.as_ref())
+        .map_err(|e| HttpError::InvalidUTF8 { source: e })?;
+
+    let es_log_lines = es_log_to_lines(lines, table, time_alias, msg_alias)
+        .map_err(|e| HttpError::ParseESLog { source: e })?;
+    Ok(es_log_lines)
+}
+
+fn try_parse_es_req(req: &Bytes) -> Result<Vec<ESLog>, HttpError> {
+    let mut eslogs = vec![];
+
+    let lines = simdutf8::basic::from_utf8(req.as_ref())
+        .map_err(|e| HttpError::InvalidUTF8 { source: e })?;
+
+    let json_chunk: Vec<&str> = lines.trim().split('\n').collect();
+    let mut i = 0;
+    let n = json_chunk.len();
+
+    if n % 2 != 0 {
+        println!("error syntax: {:?}", req);
+        return Err(HttpError::ParseESLog {
+            source: protocol_parser::ESLogError::InvaildSyntax,
+        });
+    }
+
+    while i < n {
+        let command: Command = serde_json::from_str(json_chunk[i])
+            .map_err(|e| HttpError::ParseESLogJson { source: e })?;
+        let fields: Fields = serde_json::from_str(json_chunk[i + 1])
+            .map_err(|e| HttpError::ParseESLogJson { source: e })?;
+
+        eslogs.push(ESLog { fields, command });
+
+        i += 2;
+    }
+
+    Ok(eslogs)
+}
+
+async fn coord_write_eslog(
+    coord: &CoordinatorRef,
+    tenant: &str,
+    db: &str,
+    table: &str,
+    es_log: Vec<ESLog>,
+    time_alias: Option<String>,
+    msg_alias: Option<String>,
+    span_context: Option<&SpanContext>,
+) -> Result<warp::reply::Json, HttpError> {
+    let mut span_recorder = SpanRecorder::new(span_context.child_span("write points"));
+    let mut res: HashMap<String, String> = HashMap::new();
+
+    let table_exist = {
+        if let Some(meta) = coord.meta_manager().tenant_meta(tenant).await {
+            meta.get_table_schema(db, table).is_ok()
```

Comment:
This method always returns ok, which can be checked with the is_some method: meta.get_table_schema(db, table).unwrap().is_some()